pkg/genevalogging/genevalogging.go

```diff
@@ -548,4 +566,223 @@ func (g *genevaLogging) CreateOrUpdate(ctx context.Context) error {
 			},
 		},
 	})
+
+	if err != nil {
+		return err
+	}
+
+	return g.applyDaemonSet(&appsv1.DaemonSet{
```

Comment:
Let's not end up with two DaemonSets and two mdsds running on the master, because mdsd is a memory hog.  Better approaches would be one of:
- One DaemonSet for masters, one for !masters; master DS has an additional fluentd container running for audit log forwarding
or
- (perhaps not as good) put the additional fluentd container everywhere and presumably on !masters it's a no-op because the audit directory doesn't exist

Reply:
One daemonset works, fluentd-audit on worker nodes doesn't pick up any input files, but it runs without logging errors.